backend/service/authn/crypto.go

```diff
@@ -0,0 +1,64 @@
+package authn
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"fmt"
+)
+
+// AES-256 encryption.
+type cryptographer struct {
+	gcm cipher.AEAD
+}
+
+func newCryptographer(passphrase string) (*cryptographer, error) {
+	if len(passphrase) == 0 {
+		return nil, fmt.Errorf("passphrase was empty")
+	}
+
+	key := sha256.Sum256([]byte(passphrase))
```

Comment:
Should we use bcrypt instead of sha256 hashing here? From my understanding it's slower but more secure

Reply:
we're just ensuring entropy here from the passphrase i.e. we're not storing a hash so bcrypt is not useful here.

Reply:
added comment explaining use of sha256